module1.js

```diff
@@ -17,11 +17,14 @@
 */
 
 function question1(name1) {
-  if (name1) {
+  if (name1 === "Code Your Future") {
     // TODO:
-    return;
+    return true; 
+  } else {
+    return false;
   }
-}
+  }
+
```

Comment:
Good. You could also use the rather shorter
```
return ( name1 === 'Code Your Future');
```
which will give either true or false